tools/c7n_tencentcloud/c7n_tencentcloud/resources/security_group.py

```diff
@@ -200,3 +201,45 @@ class IPPermissionEgress(SGPermission):
             'CidrV6': {}
         },
         'required': ['type']}
+
+
+@SecurityGroup.filter_registry.register('used')
+class StatisticsFilter(Filter):
+    """statistics
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+        - name: used
+          resource: tencentcloud.security-group
+          description: security group used statistical
+          filters:
+            - type: used
+            - type: value
+              key: c7n:CVM
+              op: greater-than
+              value: 0
+    """
+
+    schema = type_schema('used')
+
+    def process(self, resources, event=None):
+        cli = self.manager.get_client()
+        # DescribeSecurityGroupAssociationStatistics Maximum support 100
+        for batch in chunks(resources, 50):
+            ids = []
+            for r in batch:
+                ids.append(r['SecurityGroupId'])
+
+            resp = cli.execute_query("DescribeSecurityGroupAssociationStatistics",
+                                     {"SecurityGroupIds": ids})
+            statistics = resp["Response"]["SecurityGroupAssociationStatisticsSet"]
+            for r in batch:
+                for stat in statistics:
+                    if stat["SecurityGroupId"] == r["SecurityGroupId"]:
+                        r["c7n:CVM"] = stat["CVM"]
```

Comment:
its unclear why these three values are special. the response has additional counters under numbers a InstanceStatistics key which seem to be more comprehensive.